jans-cedarling/bindings/cedarling_go/cedarling_test.go

```diff
@@ -14,7 +14,7 @@ var bootstrapConfig string = `
         "CEDARLING_WORKLOAD_AUTHZ": "enabled",
         "CEDARLING_JWT_SIG_VALIDATION": "disabled",
         "CEDARLING_JWT_STATUS_VALIDATION": "disabled",
-        "CEDARLING_ID_TOKEN_TRUST_MODE": "none",
+        "CEDARLING_ID_TOKEN_TRUST_MODE": "never",
```

Comment:
Since we're changing a known configuration value, this needs to be documented in https://github.com/JanssenProject/jans/blob/main/docs/cedarling/cedarling-properties.md. Furthermore, the quickstart guides in the same directory also need to be updated.